packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -45,7 +45,25 @@ const _allInstances: {
   [key: string]: FocusZone;
 } = {};
 
-const _outerZones: Set<FocusZone> = new Set();
+const outZones = {
+  _outerZones: new Map<Window, Set<FocusZone>>(),
+  register(window: Window, FZ: FocusZone) {
```

Comment:
Have register return a count of active outer zones. This can be used by the caller to know if this is the first outer zone registered.

Reply:
Done